BNomad/View/ProfileView/CalendarViewController.swift

```diff
@@ -233,15 +242,34 @@ extension CalendarViewController: UICollectionViewDelegate {
             }
             cell.configureLabel(text: self.calendarDateFormatter.days[indexPath.item])
             
+            //주말 텍스트 색 설정
             if indexPath.item%7 == 0 || indexPath.item%7 == 6 {
                 cell.setWeekendColor()
             }
             
+            //오늘 텍스트 색 설정
+            if indexPath.item - calendarDateFormatter.getStartingDayOfWeek(addedMonth: monthAddedMemory) + 1 == Contents.getTodayDate()[2]
+                && Contents.getTodayDate()[1] == calendarDateFormatter.getTargetMonth(addedMonth: monthAddedMemory){
```

Comment:
로직이 조금 이해하기 어렵게 짜여 있는것 같네여,,
```suggestion
let startDay = calendarDateFormatter.getStartingDayOfWeek(addedMonth: monthAddedMemory)
            if indexPath.item - startDay + 1 == Contents.getTodayDate()[2]
                && Contents.getTodayDate()[1] == startDay {
```
중복되는 변수를 빼주는게 좋아보이긴 하지만, 그럼에도 조금 직관적으로 이해하긴 어렵네요,,
추후에 로직 수정 해야 할 것 같습니다.

Reply:
추후에 로직 수정토록 하겠습니다!! 일단 수정해주신것처럼 변수들 빼주었고, getTodayDate 함수 반환값을 딕셔너리로 만들어 가독성을 개선했습니다.